docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,198 @@
+sidebar_label: Contributors Guide to OpenSauced
+keywords:
+  - contributors
+  - guides
+---
+
+## Introduction to OpenSauced for Contributors
+
+OpenSauced is a platform dedicated to empowering you, the contributor, in your open source journey. With OpenSauced, you can find repositories to contribute to, track your contributions, gain insights into your favorite open source projects, and find new contributors to connect with.
+
+## Getting Started
+
+To get started, you will need to create an account on OpenSauced. You can do this by visiting the [app.opensauced.pizza](https://app.opensauced.pizza) and clicking the "Connect with GitHub" button. You will be prompted to sign in with your GitHub account. Once you have signed in, you will be redirected to the OpenSauced dashboard. A great place to get started is by creating your profile.
+
+## Creating Your Profile
+
+Creating your profile is a great way to showcase your open source contributions, connect with other contributors and maintainers, and share during a job search. Here are some steps to help you get started:
+
+1. After creating your account, navigate to "Settings".
+   ![settings](../../static/img/settings.png)
+
+2. Once there, you will be asked to add your name, bio, links to your social media accounts, timezone, and other forms of contact information.
+   :::tip
+   Don't forget to add your personal site, GitHub Sponsors URL, and LinkedIn so people can connect with you and learn more about your work.
+   :::
+3. In the "Interests" section, select the various programming languages and topics you are interested in. Setting them helps us recommend open source projects that fit your interests.
+4. Manage your email preferences.
+
+### Your Profile
+
+Your profile is a great resource for showcasing your contributions and sharing your open source work with others.
+
+![your profile](../../static/img/profile.png)
+
+On your profile, you will see the "Highlights" section, which you can learn more about [in this section](#highlights-connecting-and-recognizing-contributions), and three other tabs:
+
+- **Contributions**: This tab includes a graph showing the number of pull requests (PRs) you have opened, [the velocity of your PRs](../welcome/glossary.md#pr-velocity), and the number of repositories that you have contributed to. Underneath the graph, there is a "Latest PRs" chart, which shows a list of current and recent PRs. This tab is helpful if you want to showcase a steady and consistent open source contribution experience.
+- **Connections**: For pro accounts, you can reach out to other contributors and maintainers to collaborate on projects or to learn from them.
+- **Recommendations**: This tab includes a list of open source projects that are chosen based on the programming languages you picked in the "Interests" section of your public profile. It is useful if you're looking for new projects to contribute to.
+
+## Explore: Finding Your Next Open Source Project
```

Comment:
This title previously was "Explore Tab: Finding Your Next Open Source Project".
I removed the word "tab" for consistency with other tabs.